parlai/mturk/core/worlds.py

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) 2017-present, Facebook, Inc.
+# All rights reserved.
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree. An additional grant
+# of patent rights can be found in the PATENTS file in the same directory.
+from parlai.core.worlds import World, validate
+
+class MTurkWorld(World):
+    """
+    Generic world for MTurk
+    """
+    def __init__(self, mturk_agent):
```

Comment:
ideally this should look more like its parent's API, so maybe add opt back in at least?